website/docs/cloud-docs/api-docs/run.mdx

```diff
@@ -553,7 +555,16 @@ curl \
       }
     },
     {...}
-  ]
+  ],
+  "meta": {
+    "pagination": {
+      "current-page": 1,
+      "next-page": 2,
+      "prev-page": null,
```

Comment:
On line 554, there there is a "links" section. We could also reinforce the flexibility of the existing data by including the pagination links (not this example in particular but something similar)

This exists on workspace runs.. does it also exist on org runs?

For example:

```
"links": {
    "self": "https://app.terraform.io/api/v2/workspaces/ws-SqohByt1nDRfQhzT/runs?filter%5Boperation%5D=plan_only&page%5Bnumber%5D=1&page%5Bsize%5D=20",
    "first": "https://app.terraform.io/api/v2/workspaces/ws-SqohByt1nDRfQhzT/runs?filter%5Boperation%5D=plan_only&page%5Bnumber%5D=1&page%5Bsize%5D=20",
    "prev": null,
    "next": null,
    "last": "https://app.terraform.io/api/v2/workspaces/ws-SqohByt1nDRfQhzT/runs?filter%5Boperation%5D=plan_only&page%5Bnumber%5D=1&page%5Bsize%5D=20"
},
```

Reply:
Hadn't considered including this metadata portion from the original pagination method. Currently the related PR for the org runs endpoint pagination update doesn't have these links. I'm wondering if it's worth adding them now, though I suppose it wouldn't be a breaking change if the new pagination method was later updated if there was a found need for it.